ratis-server/src/main/java/org/apache/ratis/server/impl/RaftServerImpl.java

```diff
@@ -642,9 +643,24 @@ GroupInfoReply getGroupInfo(GroupInfoRequest request) {
     final RaftConfigurationProto conf =
         LogProtoUtils.toRaftConfigurationProtoBuilder(getRaftConf()).build();
     return new GroupInfoReply(request, getCommitInfos(), getGroup(), getRoleInfoProto(),
-        dir.isHealthy(), conf);
+        dir.isHealthy(), conf, getLogInfo());
   }
 
+  LogInfoProto getLogInfo(){
+    long currentTerm = getInfo().getCurrentTerm();
+    LogInfoProto.Builder logInfoBuilder = LogInfoProto.newBuilder();
+    logInfoBuilder.setApplied(TermIndex.valueOf(currentTerm, getInfo().getLastAppliedIndex()).toProto());
+    logInfoBuilder.setCommitted(getRaftLog().getLastEntryTermIndex().toProto());
```

Comment:
This is the last entry, not the committed entry.  We should use `RaftLog.getLastCommittedIndex()`.

Reply:
yeah that makes sense! Thanks Nicholas for catching that!

Reply:
fixed!